setup: flavors

drake-ho-goldenowl opened the pull request:
## Todo
- [X] Set up flavors
- [X] Write [document: How to change google-service](https://github.com/GoldenOwlAsia/mobile-flutter-template/wiki/Flavors)

## Screenshot
<img src="https://github.com/GoldenOwlAsia/mobile-flutter-template/assets/104961439/b53b05ae-13e3-4b5f-98ab-4963b21285a5" width=30% height=30%> <img src="https://github.com/GoldenOwlAsia/mobile-flutter-template/assets/104961439/441f986f-4b26-4753-9db2-edd07f77ce53" width=30% height=30%>

drake-ho-goldenowl committed:
Set up flavors

drake-ho-goldenowl committed:
Set up flavors

jimmy-to-goldenowl commented:
Hi @drake-ho-goldenowl. Thanks a lot for your PR. 
Please help update the readme. how to run staging and prod. and we should push the file .vscode/launch.json for run for earch lavors too

drake-ho-goldenowl committed:
Update readme

add launch.json file
Fix conflict

drake-ho-goldenowl committed:
Merge branch 'setup/flavors' of https://github.com/GoldenOwlAsia/mobile-flutter-template into setup/flavors

drake-ho-goldenowl committed:
Fix conflict

drake-ho-goldenowl commented:
> Hi @drake-ho-goldenowl. Thanks a lot for your PR. Please help update the readme. how to run staging and prod. and we should push the file .vscode/launch.json for run for earch lavors too

I have added the file .vscode/launch.json and updated Readme: how to run staging and prod in a new commit. I also fix conflicts with development and test success building ios, android.

drake-ho-goldenowl committed:
Update README.md

jimmy-to-goldenowl committed:
Feat/chat (#16)

* feat: chat

* fix

* update pod lock

---------

Co-authored-by: j1mmyto9 <khanhuitse05@gmail.com>

khanhuitse05 committed:
Merge branch 'development' into setup/flavors
